### PR TITLE
Update alert message and fix typo error

### LINKED ISSUE #151: error message refers to non-existent file
The message refers to `kolibri.log`:

image

However the files changed somewhat recently to `kolibri.txt` and `debug.txt` as described here:

https://kolibri.readthedocs.io/en/latest/manage/get_support.html#locate-kolibri-log-files

Let's update the ~docs~ modal to reflect the new log files.

### PR DESCRIPTION
**Pull request by mrpau-richard**
I updated the alert message of the log file path.
here's the sample screenshoot.
Screen Shot 2020-02-05 at 2 41 07 PM

This fixes #151 #142

### CODE REVIEW

File: src/installer-and-application-po-files/kolibri-application.po [hunk 1]
```diff
@@ -94,7 +94,7 @@ msgstr ""
 msgid "Installation in progress."
 msgstr ""
 
-msgid "Kolibri failed to start. Additional information can be found in the log file:"
+msgid "Kolibri failed to start. Additional information can be found in the log directory:"
```
Comment (cpauya) [inline src/installer-and-application-po-files/kolibri-application.po]: Maybe remove the `log` and put a space after the color so it reads: `... found in the directory:<space>`?

File: src/gui-source/Kolibri/Kolibri.cpp [hunk 2]
```diff
@@ -109,7 +109,7 @@ void kolibriScriptPath(char *buffer, const DWORD MAX_SIZE)
 }
 
 void kolibriFailedMessage() {
-	MessageBox(HWND_DESKTOP, concatenateWchar(getStr(ID_STRING_21_en), getWC(getKolibriHomePath()), getWC("\\logs\\kolibri.log")), L"Kolibri", MB_OK | MB_ICONINFORMATION);
+	MessageBox(HWND_DESKTOP, concatenateWchar(getStr(ID_STRING_21_en), getWC(getKolibriHomePath()), getWC("\\logs")), L"Kolibri", MB_OK | MB_ICONINFORMATION);
```
Comment (cpauya) [inline src/gui-source/Kolibri/Kolibri.cpp]: Maybe add a back-slash at the end to denote it's a directory?  Like `...\logs\` instead?

**Comment by mrpau-richard** [comment]
Thanks for reviewing this @cpauya 

Here's the new updated alert message.
Screen Shot 2020-02-10 at 4 34 20 PM

**Comment by indirectlylit** [comment]
thank you!

**Comment by radinamatic** [comment]
@mrpau-richard The `D` drive is not hardcoded, right? The drive letter is extracted by the script from where the `./kolibri` folder is located?

I suggest a slight change in the wording of the alert message:

```
Kolibri is unable to start. You can find additional information in the X:\.kolibri\logs folder.
```

**Comment by mrpau-richard** [comment]
> @mrpau-richard The D drive is not hardcoded, right? 

Your right @radinamatic that's not hardcoded. I used the message you suggested and I uploaded a sample alert Window of it.
Screen Shot 2020-02-11 at 3 45 10 PM

**Comment by mrpau-richard** [comment]
Thanks @radinamatic , I think this PR is ready to merge now.

**Comment by indirectlylit** [comment]
thanks all. please remember to set the milestone on PRs and issues

**Pull request merged**